ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -1173,6 +1184,11 @@ vec3 textureOrRGB19E7(in uvec2 data, in bool texPresenceFlag, in mat2 dUV)
 #define OP_BUMPMAP			13u
 #define OP_BLEND			12u
 
+bool opcode_isRough(in uint op)
+{
+	return op==OP_ROUGHDIFFUSE || op==OP_ROUGHDIELECTRIC || op==OP_ROUGHCONDUCTOR || op==OP_ROUGHPLASTIC || op==OP_WARD || op==OP_ROUGHCOATING;
```

Comment:
optimize later, either a bit flag or `<`+`>=` compare, this 5 comparison thing is awful